### ISSUE TITLE
【结业答辩】内容安排

### ISSUE THREAD

**Issue by wqw547243068** [wqw547243068/school-of-ai-beijing#13]
## 结业安排
5月25日，中文分支的5次课程结束，接下来是验收环节
- 时间：两周后的**6月16日下午17：00-19：00**
- 地点：老地方，**清华数据科学研究院**
- 内容：大家任选主题，（单选、多选不限），中英文分支一起汇报
- 分组：根据时间、主题，我们会做适当调整
- 评估：平时作业+现场表现（权重60%：40%），根据最终得分，决定是否符合毕业资质，颁发证书

### 选题
- 请大家于本周五（5月31日）前，完成报名及选题
   - 报名方式：本issue下回复即可
- 格式
> - 汇报人：小明，github账号，线下班/线上班，参加3次课程
> - 平时作业：必选三道（持续更新链接）+ 可选（非必须，持续更新链接）
> - 汇报主题：学习总结/专题报告-学习率优化方案/Kaggle比赛

### 平时作业
- 平时作业：中文系列的5次课中，布置了很多作业，不要求都完成：
   - [x] 【必选】宠物分类实践：动手实践，最好有自己的优化
   - [x] 【必选】机器学习专题：熟记机器学习基本概念、流程，完成iris分类、回归预测案例，同时拆解第一课的宠物分类代码，看看有哪些环节缺失，提出自己的见解
   - [x] 【必选】神经网络专题：神经网络基本构成，用tf、pytorch、keras实现mnist分类（如果是别的项目更好）
    - [x] 【可选】CNN与CV：CNN基本流程，CV应用点，想办法去优化宠物分类准确率
    - [ ] 【可选】RNN与NLP：RNN基本流程，NLP应用点，文本挖掘实战，聊天机器人实战
- 注：以上作业资料甚至代码都在ppt、主页里
### 汇报主题
- 主要有三类，大家任选一个，鼓励多选
- 按照难易程度递增如下：
   - [x] （1）**学习总结**：汇总历次课程的心得，总结到自己的github/ppt，一定要是自己写的，别copy，目的是让大家构建自己的深度学习知识体系，横向打基础
   - [x] （2）**专题报告**：根据课程中遇到的问题，选一个技术点，深入研究，整理出来（github/ppt），给大家讲解，这部分属于深度上的扩展
   - [x] （3）**Kaggle比赛**：去kaggle选一个题目，尝试解决，名次最好能达到top 30%，同时整理自己的参赛经过
      - 注意：选择kaggle的人，请关注这个[帖子](https://github.com/wqw547243068/school-of-ai-beijing/issues/1)，以及：
         - [kaggle数据挖掘比赛经验分享](https://www.toutiao.com/a6694125269071954435/?tt_from=android_share&utm_campaign=client_share&timestamp=1558660207&app=news_article_lite&utm_medium=toutiao_android&req_id=20190524091006010023075150760AB41&group_id=6694125269071954435)
         - [分分钟带你杀入kaggle top 1%](https://zhuanlan.zhihu.com/p/27424282)
         - [kaggle求生：亚马逊热带雨林](https://zhuanlan.zhihu.com/p/28084438)
### 备注

> - 希望大家好好准备
> - 学习是自己的事情，越是认真对待，学到的越多，对自己负责，加油！

![image](https://user-images.githubusercontent.com/3257257/58416940-c55aaa00-80b5-11e9-9699-a3451c3791a9.png)

**Comment by ztq222** [comment]
汇报人：张天奇，ztq222，线下班，参加4次课程
平时作业：必选三道 + CNN与CV（不一定能做出来）
汇报主题：Kaggle比赛

**Comment by huakaibanmu** [comment]
汇报人:王瑞华，huakaibanmu，线下班，参加线下三次课程（另线上两次）
平时作业：必选三道+RNN与NLP
汇报主题：kaggle比赛

**Comment by daaafuuu** [comment]
汇报人:黄安付，daaafuuu，线下班，参加线下后三次课程
平时作业：必选三道+RNN与NLP
汇报主题：专题报告

---
可否注明下专题名称？
---
结合深度学习花书优化算法章节内容基础，分享对fit_one_cycle的理解。

**Comment by zhouqihong** [comment]
汇报人：蒋宝尚，jiangbaosahng, 线下班，参加5次课程
平时作业：必选三道 + RNN与NLP（不一定能做出来）
汇报主题：（1）学习总结

汇报人：刘毅然，jllstone，线下班/线上班，参加2次线上3次线下课程
平时作业：必选三道+ 可选CNN与CV
汇报主题：学习总结

汇报人：谭永川，from0to8，线下班/线上班，参加1次线上2次线下课程
平时作业：必选三道+ 可选CNN与CV
汇报主题：学习总结

汇报人:王瑞华，huakaibanmu，线下班，参加线下三次课程（另线上两次）
平时作业：必选三道+RNN与NLP
汇报主题：学习总结+kaggle比赛(尝试)

汇报人：周启红，zhouqihong, 线下班，参加5次课程
平时作业：必选三道 + RNN与NLP（不一定能做出来）
汇报主题：（1）学习总结

**Comment by LiJinghua18** [comment]
汇报人：李婧华，LiJinghua18，线下班，参加3次线下课，1次线上课
平时作业：必选三道 + 可选CNN与CV
汇报主题：学习总结

**Comment by xc2017** [comment]
汇报人：邢畅，xc2017，线下班，参加3次线下课程
平时作业：必选三道+ 可选CNN与CV
汇报主题：学习总结

**Comment by wqw547243068** [comment]
# 介绍 Introduction
- 时间：2019-06-16
- 中文分支的往期内容及课件，见[地址](https://wqw547243068.github.io/school-of-ai-beijing/)
- 答辩现场，由于学员较多，时间长，从16:00-19:40，衷心感谢各位坚持到最后
![image](https://user-images.githubusercontent.com/3257257/59566552-55c45300-9094-11e9-8eec-f23736653eaa.png)
![image](https://user-images.githubusercontent.com/3257257/59566558-61177e80-9094-11e9-88de-7e9ff85ea939.png)
   - [Clear image](https://github.com/wqw547243068/school-of-ai-beijing/blob/master/other/soai_presentation.jpg?raw=true),[Clear image](https://github.com/wqw547243068/school-of-ai-beijing/blob/master/other/soai_presentation1.jpg?raw=true)
# 中文分支 Chinese branch
1. 第一位 邢畅, xingchang, 16:07，学习总结，在地理信息系统里的应用
   - 介绍了GIS领域的应用，整理了详细的学习笔记、脑图
1. 第二位 张天奇，Tianqi.Zhang，16:27，kaggle比赛，基于新闻语料的股价涨跌预测，Eric Zhang
   - 成绩：356/2800, ≈13%，4个人的团队，负责特征工程+多线程实现，xgboost主要负责分类
   - 提问：
      - xGBoost的原理，既可以用于分类，还可以用于回归
      - 代码里将的metric用的是MSE，而xgb导入的是分类器，看似矛盾？
1. 第三位 五个人，推荐资料：数学基础张宇36讲，课程内容回顾；
   - 蒋宝尚，mnist的python实现，文本分析（大爆炸+华为+五月天）
   - 周启红，学术文章关键词提取，学习过程中遇到的坑儿
   - 王瑞华，视频汇报
1. 第四位  黄安付，17:15，专题报告：fit_one_cycle()详解
   - 优化算法，鞍点（saddle point），病态矩阵，粒子群算法
   - 提问：动量解释，附加SGD系列优化算法演进，[An overview of gradient descent optimization algorithms](http://ruder.io/optimizing-gradient-descent/),[How to Escape Saddle Points Efficiently](http://www.offconvex.org/2017/07/19/saddle-efficiency/),[SGD算法比较](https://blog.slinuxer.com/2016/09/sgd-comparison)，如下图，注意：v是动量部分，g是梯度部分![](https://raw.githubusercontent.com/SwordYork/simplified-deeplearning/master/sgd-comparison/figures/relation.png) ![](http://ruder.io/content/images/2016/09/contours_evaluation_optimizers.gif) ![](http://ruder.io/content/images/2016/09/saddle_point_evaluation_optimizers.gif)
1. 第五位  李靖华，17:36，学习总结
   - git，学习方法总结
   - 实习经历，4w张图片分类，用代码1天完成，做事情时先别急着动手
   - 成功拿到创新工场的实习机会，新人到职场的华丽蜕变
# 英文分支 English branch
- 张子涵, I wrote a short article with some doodling on neural networks and how they work. Here is the link if you want to check it out [medium文章](https://medium.com/@milena.lazzaretti5/mathematical-aspects-of-deep-learning-graphically-explained-e05c6a5acbfa)
- Mohamed Amine Aguir的[Convolutional Neural Network](https://www.linkedin.com/pulse/deep-learning-convolutional-neural-network-cnn-mohamed-amine-aguir)
- Nabil Affo的Uber ride评论分类项目，[jupyter代码](https://github.com/nabil2i/part1_fastai/blob/master/Uber.ipynb), Medium文章[Loss functions in machine learning and deep learning](https://medium.com/@affo.nabil/loss-functions-in-machine-learning-and-deep-learning-846446b87d5b)
- Alexander R Kivaisi, Categorizing Soccer Teams（代码见资料汇总区）, Medium文章[Data Augmentation](https://medium.com/@regak2010/data-augmentation-85e8d4497578)
- Min Wang(汪敏)
- Yuanyuan，CFAIR-10图像分类
# 资料汇总 Resources
- 部分学员的答辩整理已收集到[目录](https://github.com/wqw547243068/school-of-ai-beijing/tree/master/homework), 欢迎大家积极分享自己的资料，共同学习，添加方式，回复本帖
- Part of the resources have been collected to this [address](https://github.com/wqw547243068/school-of-ai-beijing/tree/master/homework), it's recommended to share your own ppt/code/post here by reply bellow, learn from each other
- 大部分基础知识以及扩展资料都在之前的课程ppt中,多看SoAI主页[The School of AI Beijing](https://wqw547243068.github.io/school-of-ai-beijing)，**毕业了并不代表学完了，或者真的学好了，请大家有空多温习，少走弯路**
